packages/seacas/libraries/ioss/src/text_mesh/Iotm_TextMesh.h

```diff
@@ -0,0 +1,366 @@
+        {"TRI_3_2D",     TopologyMapEntry(Ioss::Tri3::name,        {false,false,true ,false})},
+        {"TRI_4_2D",     TopologyMapEntry(Ioss::Tri4::name,        {false,false,true ,false})},
+        {"TRI_6_2D",     TopologyMapEntry(Ioss::Tri6::name,        {false,false,true ,false})},
+        {"QUAD_4_2D",    TopologyMapEntry(Ioss::Quad4::name,       {false,false,true ,false})},
+        {"QUAD_8_2D",    TopologyMapEntry(Ioss::Quad8::name,       {false,false,true ,false})},
+        {"QUAD_9_2D",    TopologyMapEntry(Ioss::Quad9::name,       {false,false,true ,false})},
+        {"SHELL_TRI_3",  TopologyMapEntry(Ioss::TriShell3::name,   {false,false,false,true })},
+        {"SHELL_TRI_4",  TopologyMapEntry(Ioss::TriShell4::name,   {false,false,false,true })},
+        {"SHELL_TRI_6",  TopologyMapEntry(Ioss::TriShell6::name,   {false,false,false,true })},
+        {"SHELL_QUAD_4", TopologyMapEntry(Ioss::Shell4::name,      {false,false,false,true })},
+        {"SHELL_QUAD_8", TopologyMapEntry(Ioss::Shell8::name,      {false,false,false,true })},
+        {"SHELL_QUAD_9", TopologyMapEntry(Ioss::Shell9::name,      {false,false,false,true })},
+        {"TET_4",        TopologyMapEntry(Ioss::Tet4::name,        {false,false,false,true })},
+        {"TET_8",        TopologyMapEntry(Ioss::Tet8::name,        {false,false,false,true })},
+        {"TET_10",       TopologyMapEntry(Ioss::Tet10::name,       {false,false,false,true })},
+        {"TET_11",       TopologyMapEntry(Ioss::Tet11::name,       {false,false,false,true })},
+        {"PYRAMID_5",    TopologyMapEntry(Ioss::Pyramid5::name,    {false,false,false,true })},
+        {"PYRAMID_13",   TopologyMapEntry(Ioss::Pyramid13::name,   {false,false,false,true })},
+        {"PYRAMID_14",   TopologyMapEntry(Ioss::Pyramid14::name,   {false,false,false,true })},
+        {"WEDGE_6",      TopologyMapEntry(Ioss::Wedge6::name,      {false,false,false,true })},
+        {"WEDGE_12",     TopologyMapEntry(Ioss::Wedge12::name,     {false,false,false,true })},
+        {"WEDGE_15",     TopologyMapEntry(Ioss::Wedge15::name,     {false,false,false,true })},
+        {"WEDGE_18",     TopologyMapEntry(Ioss::Wedge18::name,     {false,false,false,true })},
+        {"HEX_8",        TopologyMapEntry(Ioss::Hex8::name,        {false,false,false,true })},
+        {"HEX_20",       TopologyMapEntry(Ioss::Hex20::name,       {false,false,false,true })},
+        {"HEX_27",       TopologyMapEntry(Ioss::Hex27::name,       {false,false,false,true })}
+    };
+  }
+};
+
+using Topology = TopologyMapEntry;
+using TextMeshData = text_mesh::TextMeshData<int64_t, TopologyMapEntry>;
+using ElementData = text_mesh::ElementData<int64_t, TopologyMapEntry>;
+using Coordinates = text_mesh::Coordinates<int64_t, TopologyMapEntry>;
+using TextMeshParser = text_mesh::TextMeshParser<int64_t, IossTopologyMapping>;
+using MapVector = std::vector<int64_t>;
```

Comment:
This is the same as an `Ioss::Int64Vector`.  Since we use the `Ioss::IntVector` below, we should probably be consistent and use `Ioss::Int64Vector` instead of `MapVector` unless there is something I am overlooking.  Was just confusing when I was looking at functions below and wanting to know the difference between them...

This probably came from the Iogn_GeneratedMesh originally which should also be updated...

Reply:
Yup, this was from Iogn_GeneratedMesh ... I'll remove the MapVector definition

Reply:
I will try to fix elsewhere...